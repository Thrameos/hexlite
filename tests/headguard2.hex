d(1). d(2).
f(1).
q(X) : f(Y) :- d(X).
