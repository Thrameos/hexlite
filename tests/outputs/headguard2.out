{d(1),d(2),f(1),q(1),q(2)}
